pom.xml

```diff
@@ -121,6 +121,7 @@
         <bt-hamcrest-version>2.2</bt-hamcrest-version>
         <bt-httpasyncclient-version>4.1.5</bt-httpasyncclient-version>
         <bt-httpclient-version>4.5.13</bt-httpclient-version>
+        <bt-custom-httpclient-osgi-version>4.5.13-1</bt-custom-httpclient-osgi-version>
```

Comment:
Let's remove the original httpclient-osgi, and replace it with the fork in dependencyManagement section.

Reply:
line 814.

Reply:
Good catch. Didn't see that one